prefab.proto

```diff
@@ -120,18 +150,7 @@ message LimitRequest {
   LimitDefinition.SafetyLevel safety_level = 6; // [default = L4_BEST_EFFORT];
 }
 
-message FeatureFlagVariant {
-
-  optional int64 int = 1;
-  optional string string = 2;
-  optional double double = 3;
-  optional bool bool = 4;
-
-  string name = 5;
-  string description = 6;
-}
-
-message Criteria {
+message Criterion {
   enum CriteriaOperator {
```

Comment:
Maybe this should also be `CriterionOperator`, since it only operates on a single criterion?